### PR TITLE
Fixes #24046: Add new Jenkinsfile-release

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/24046

### CODE REVIEW

File: Jenkinsfile-release [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+def job = ""
+def errors = []
+def running = []
+
+
+pipeline {
+    agent none
+
+    triggers {
+        cron('@midnight')
```
Comment (amousset) [inline Jenkinsfile-release:19]: we probably don't want it to run every day

**Comment by VinceMacBuche** [comment]
Commit modified

**Review by amousset — APPROVED** [review]

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**